### PR TITLE
Default Authfile to proper path

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@nalind @TomSweeneyRedHat @giuseppe @vrothberg @pixdrift PTAL

File: pkg/cli/common.go [hunk 1]
```diff
@@ -246,3 +247,15 @@ func VerifyFlagsArgsOrder(args []string) error {
 	}
 	return nil
 }
+
+func GetDefaultAuthFile() string {
```
Comment (TomSweeneyRedHat) [inline pkg/cli/common.go]: I'm surprised golang didn't yell at you for not having a doc line above this....

**Comment by TomSweeneyRedHat** [comment]
@rh-atomic-bot r+

**Comment by vrothberg** [comment]
I restarted the two failed jobs; network flake :roll_eyes:

**Pull request closed without merging**